src/shelly_output.cpp

```diff
@@ -60,6 +65,17 @@ Status OutputPin::SetState(bool on, const char *source) {
   return Status::OK();
 }
 
+Status OutputPin::SetStatePWM(float duty, const char *source) {
```

Comment:
I don't like this, but I could find any other solution without doing a big refactoring.